src/test/groovy/de/triplet/gradle/play/PlayPublisherPluginTest.groovy

```diff
@@ -133,6 +119,106 @@ class PlayPublisherPluginTest {
         }
     }
 
+    @Test
+    public void testJsonFileBackwardsCompatibility() {
+        Project project = TestHelper.evaluatableProject()
+
+        project.play {
+            jsonFile new File("key.json");
+        }
+
+        project.evaluate()
+
+        assertEquals("key.json", project.extensions.play.jsonFile.name)
+    }
+
+    @Test
+    public void testPlayAccountBackwardsCompatibility() {
+        Project project = TestHelper.evaluatableProject()
+
+        project.play {
+            serviceAccountEmail = 'service-account@test.com'
+            pk12File = new File("key.p12")
+        }
+
+        project.evaluate()
+
+        project.extensions.play.serviceAccountEmail == 'service-account@test.com'
+        project.extensions.play.pk12File = new File("key.p12")
+    }
+
+    @Test
+    public void testPlaySigningConfigs() {
+        Project project = TestHelper.evaluatableProject()
+
+        project.android {
+            playAccountConfigs {
+                defaultAccountConfig {
+                    serviceAccountEmail = 'default@exmaple.com'
+                    pk12File = project.file('first-secret.pk12')
+                }
+                free {
+                    serviceAccountEmail = 'first-mail@exmaple.com'
+                    pk12File = project.file('secret.pk12')
+                }
+                paid {
+                    serviceAccountEmail = 'another-mail@exmaple.com'
+                    pk12File = project.file('another-secret.pk12')
+                }
+            }
+
+            defaultConfig {
+                playAccountConfig = playAccountConfigs.defaultAccountConfig
+            }
+
+            productFlavors {
+                defaultFlavor {
+
+                }
+                free {
+                    playAccountConfig = playAccountConfigs.free
+                }
+                paid {
+                    playAccountConfig = playAccountConfigs.paid
+                }
+            }
+        }
+        project.evaluate()
+
+        assert project.tasks.bootstrapDefaultFlavorReleasePlayResources.playAccountConfig.serviceAccountEmail == 'default@exmaple.com'
```

Comment:
For the unit test we normally use the static methods from `org.junit.Assert` rather then the `assert` keyword.